_posts/2017-04-13-sketching-in-code.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Sketching in code to build online services"
+category: [blog]
+tags: 'technology'
+author: Simon Schwartz
+author-excerpt: "Simon Schwartz is a frontend developer at the Digital Transformation Agency."
+thumbnail: /images/blog-thumbnails/Sketch_in_code_sml.png
+hero-image: /images/blog-content/Sketch_in_code_hero.png
+---
+
+!Four developers working together in an office.]({{ site.url }}{{ site.baseurl }}{{ page.hero-image }})
```

Comment:
Needs a [ after the ! e.g. change to 
`![Four developers working together in an office.]({{ site.url }}{{ site.baseurl }}{{ page.hero-image }})`